web/route.go

```
@@ -33,12 +30,34 @@
 	router := gin.Default()
 	router.UseH2C = true
 
+	if cfg.Auth.Enable == true {
```

golangci-lint (1.62.2)

S1002: should omit comparison to bool constant, can be simplified to `cfg.Auth.Enable`

(gosimple)